File: src/main/java/nu/peg/slack/pt/service/LocationService.java
package nu.peg.slack.pt.service;

import nu.peg.slack.pt.api.slack.model.SlackMessage;
import nu.peg.slack.pt.model.Locations;

public interface LocationService {

    public Locations queryLocations(String from, String to);

    public SlackMessage makeRefinementMessage(Locations locations);
}

File: src/main/kotlin/nu/peg/slack/pt/service/LocationService.kt
package nu.peg.slack.pt.service

import nu.peg.slack.pt.api.slack.model.SlackMessage
import nu.peg.slack.pt.model.ConnectionRequest
import nu.peg.slack.pt.model.Locations

interface LocationService {

    fun queryLocations(from: String, to: String): Locations

    fun makeRefinementMessage(locations: ConnectionRequest): SlackMessage
}
